scripts/Vote.ts

```diff
@@ -15,7 +15,7 @@ async function voteScript(): Promise<void> {
   if (arg.length > 2) {
     throw new Error("Please only vote for one proposal");
   } else {
-    arg = arg[0];
+    arg = arg[0 - 1];
```

Comment:
Be careful here. 

arg[0 - 1] = arg[-1]. I don't know what will happen if you try to get the -1 indexed value, maybe out of bounds.

We want to get the value subtracted by one, not the index. arg = arg[0] - 1, where the sub is out of the brackets.